src/test/java/com/sfuentes/StudentTest.java

```diff
@@ -0,0 +1,92 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StudentTest {
+
+  @Test
+  void sortByWomenAndMen_TenStudents_OnlyWomen() {
```

Comment:
You didn't rename the method  in your tests methods. Please update it in all of them.

Reply:
fixed